src/tree-node/index.js

```diff
@@ -88,7 +88,7 @@ class TreeNode extends PureComponent {
     const style = keepTreeOnSearch || !searchModeOn ? { paddingLeft: `${(_depth || 0) * 20}px` } : {}
 
     return (
-      <li className={liCx} style={style} {...getDataset(dataset)}>
+      <li className={liCx} style={style} {...getDataset(dataset)} role="option" aria-selected={checked}>
```

Comment:
Perhaps we should add a function getAriaTags, {...getAriaTags(props)}. It would make it easier to check on simpleSelect since I think we need different roles depending on type of dropdown. `role="option"` is fine when emulating a regular dropdown, but when we have an expandable tree we probably need to use `role="row"` and `role="tree"` on the "parent" instead.

Then we can add `aria-level={_depth +1}` and` aria-expanded={_children && expanded}` for this also which is not valid with role="option".

We probably also should add `aria-readonly={readonly || disabled}`

Reply:
Agreed and fixed!